### PR TITLE
[Life Domains] Rework Choose Domain Flow

### PR DESCRIPTION
**Pull request by MarkDavies0**
## Changes
Changes the domain selection button to a toggle and removes the confirmation screen after each domain. Update domain details view to match the mockup. Add subdomains for physical domain. Introduce all theme and color changes from #124

## Requirements
run the setup script

## PR Dependencies
Depends on: 
#123

## Merged Branches and Other Dependencies
Merged branch(es):
#124

## Subtasks / Progress

- [x] Add real domain & strategy copy
- [x] Add subdomains capacity (only for Physical domain at this point)
- [x] Change left and right arrows to 3 arrows icon

## Testing
Go through all the domain related screens. Add and remove domains, thoroughly test the Physical subdomain selection (cancel, back, change, learn more, etc.). Make sure correct strategies are shown in details view

- [X] Manual

### CODE REVIEW

**Comment by brendeny** [comment]
Clicking through the domains flow = pass. When I click the orb on the home screen I receive an error in the android physical device app

**Comment by MarkDavies0** [comment]
@brendeny I have just tried this and had no issues. I think you need to delete your user's `focusedDomains` and `chosenStrategies` fields in firebase. Domains must be selected before tapping the orb. I have added an if statement in the orb tap handler to guard against the error in the screenshots.

File: common/utils/dateHelpers.ts [hunk 1]
```diff
@@ -264,6 +264,16 @@ export function formatFullDate(date: Date | number, withYear: boolean = false):
     return `${dayOfWeek}, ${day} ${month}`;
 }
 
+export function formatDateDayMonthYear(date: Date | number): string {
+    const d = getDate(date);
```
Comment (brendeny) [inline common/utils/dateHelpers.ts:268]: is there a clearer name we can use than d?

**Review by brendeny — APPROVED** [review]
Looks great Mark. Tested on android physical device and passes

File: common/utils/dateHelpers.ts (hunk 1 above)
Comment (MarkDavies0) [reply common/utils/dateHelpers.ts:268]: I agree a one char name is not ideal but it was done to match the style of the other functions in this file. I think it should be fine as these are only small helpers

**Comment by wynnset** [comment]
@MarkDavies0 could you please fix the merge conflicts on here and then assign back

**Pull request merged**